actix-http/src/extensions.rs

```diff
@@ -75,6 +75,19 @@ impl fmt::Debug for Extensions {
 mod tests {
     use super::*;
 
+    #[test]
+    fn test_insert() {
+        let mut map = Extensions::new();
+
+        let prev = map.insert("foo");
+        assert_eq!(prev, None);
```

Comment:
```suggestion
        assert!(prev.is_none());
```

Reply:
I prefer `assert_eq!` for `Option` checks (where applicable) because it tells you the value if the option being tested is unexpectedly `Some`.